bubuku/cli.py

```diff
@@ -85,15 +85,17 @@ def restart_broker(broker: str):
 @click.option('--empty_brokers', type=click.STRING,
               help="Comma-separated list of brokers to empty. All partitions will be moved to other brokers")
 @click.option('--exclude_topics', type=click.STRING, help="Comma-separated list of topics to exclude from rebalance")
-def rebalance_partitions(broker: str, empty_brokers: str, exclude_topics: str):
+@click.option('--step', type=click.INT, show_default=True,
```

Comment:
@lmontrieux Yep, you are completely right here. Therm step is completely wrong here, I will rename it everywhere to parallelism